frontend/app/views/tier/upgrade/freeToPaid.scala.html

```diff
@@ -1,89 +1,126 @@
+@import abtests.PayPalTestVariants
+@import com.gu.memsub.promo.{Upgrades, ValidPromotion}
 @import com.gu.memsub.subsv2.CatalogPlan.PaidMember
-@import com.gu.memsub.subsv2.{MonthYearPlans, _}
+@import com.gu.memsub.subsv2._
+@import com.gu.salesforce.Tier.{Partner, Supporter}
 @import play.api.mvc.RequestHeader
 @import play.filters.csrf
+@import views.html.helper._
 @import views.support.DisplayText._
 @import views.support.MembershipCompat._
-@import views.support.{CountryWithCurrency, PageInfo, PaidPlans}
-@import com.gu.memsub.promo.ValidPromotion
-@import com.gu.memsub.promo.Upgrades
+@import views.support.{CountryWithCurrency, IdentityUser, PageInfo}
 @(currentPlan: CatalogPlan.Member,
   targetPlans: MonthYearPlans[PaidMember],
   countriesWithCurrency: List[CountryWithCurrency],
-  userFields: com.gu.identity.play.PrivateFields,
+  idUser: IdentityUser,
   pageInfo: PageInfo,
-    validPromo: Option[ValidPromotion[Upgrades]]
+    validPromo: Option[ValidPromotion[Upgrades]],
+    paypalTestVariant : Option[String] = Some(PayPalTestVariants.withPayPal)
 )(implicit token: csrf.CSRF.Token, request: RequestHeader)
 
-@import views.html.helper._
-
 @main("Change Tier | " + targetPlans.tier.name + " from " + currentPlan.tier.name, pageInfo = pageInfo) {
```

Comment:
The margins for membership are a pain:

![margin-problem](https://cloud.githubusercontent.com/assets/5131341/23414419/cf4b9b08-fdd3-11e6-9e98-3d7e1cebca42.png)

I added a fix for this to the main template, if you pass `margins=false` in to main here it should get rid of them.

Do you think it's worth using the simple header here too (`simpleHeader=true`)?